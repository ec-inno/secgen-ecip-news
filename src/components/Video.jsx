import React from 'react';
import { useTranslation } from 'react-i18next';

import image from '../components/assets/images/video-image.png';

const Video = () => {
  const { t } = useTranslation();

  return (
    <figure className="ecl-media-container">
      <img
        className="ecl-media-container__media"
        src={image}
        alt={t("The European Citizens' Initiative in video.")}
      />
      <figcaption className="ecl-media-container__caption">
        {t("The European Citizens' Initiative in video.")}
      </figcaption>
    </figure>
  );
};

export default Video;
