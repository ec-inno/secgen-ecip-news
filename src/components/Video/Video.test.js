import React from 'react';
import renderer from 'react-test-renderer';
import Video from './Video';

describe('Video', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Video />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
