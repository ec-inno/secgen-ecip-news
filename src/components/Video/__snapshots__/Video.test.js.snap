// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video renders correctly 1`] = `
<figure
  className="ecl-media-container"
>
  <img
    alt="The European Citizens' Initiative in video."
    className="ecl-media-container__media"
    src="file"
  />
  <figcaption
    className="ecl-media-container__caption"
  >
    The European Citizens' Initiative in video.
  </figcaption>
</figure>
`;
